### PR TITLE
front-end changes 

### PR DESCRIPTION
**Pull request by Gangotri-sg3**

### CODE REVIEW

File: client/src/components/Admin/Products/AddForm.js [hunk 1]
```diff
@@ -4,79 +4,80 @@ import { useEffect, useState } from "react";
...
-    imageData.append("type",  details.type);
-    imageData.append("category",  categoryid);
-    
-    if(Object.keys(details).length == 0){
+    imageData.append("price", details.price);
+    imageData.append("description", details.description);
+    imageData.append("size", details.size);
+    imageData.append("color", details.color);
+    imageData.append("type", details.type);
+    imageData.append("category", categoryid);
```
Comment (iMagesh) [inline client/src/components/Admin/Products/AddForm.js:28]: Can we append all these in one line? to simplify

File: client/src/components/Admin/Products/UserList.js [hunk 2]
```diff
@@ -74,91 +71,91 @@ const UserList = ({}) => {
...
-                                    {display.email}
-                                  </li>
-                                </ul>
+                      {display.map((display) => (
+                        <tr class="candidates-list">
+                          <td class="title">
+                            <div class="thumb">
+                              <img
+                                class="img-fluid"
+                                src="https://bootdey.com/img/Content/avatar/avatar7.png"
```
Comment (iMagesh) [inline client/src/components/Admin/Products/UserList.js:80]: can remove this image and simple use a placeholder avatar from fontawesome or bootstrap?

File: client/src/components/Admin/Products/UserList.js [hunk 3]
```diff
@@ -3,13 +3,12 @@ import { useNavigate } from "react-router-dom";
 import axios from "../../../utils/ajax-helper";
 import ErrorMessages from "./ErrorMessages";
 import Pagination from "../Categories/Pagination";
```
Comment (iMagesh) [inline client/src/components/Admin/Products/UserList.js:5]: Can we move this Pagination component to the Common folder?

**Comment by iMagesh** [comment]
@Gangotri-sg3 you have more comments :D

File: client/src/components/Admin/Products/UserList.js (hunk 3 above)
Comment (Gangotri-sg3) [reply client/src/components/Admin/Products/UserList.js:5]: I moved

**Pull request merged**